### PR TITLE
[Artwork View] PURCHASE-1485 - Analytics fixes and updates

### PR DESCRIPTION
**Pull request by kierangillen**
- https://artsyproduct.atlassian.net/browse/PURCHASE-1485
- Schema: https://docs.google.com/spreadsheets/d/1P6qUe7yFPzly5JYicAa6eQ3BAy9s7Bx1_6k-Q5hB-Pw/edit#gid=868762451

#trivial
#skip_new_tests

### CODE REVIEW

File: src/lib/AppRegistry.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from "react"
 import { AppRegistry, View, YellowBox } from "react-native"
 import { Container as RelayContainer } from "react-relay"
 
+import { SafeAreaInsets } from "lib/types/SafeAreaInsets"
```
Comment (sweir27) [inline src/lib/AppRegistry.tsx]: Curious why this was added?

File: src/lib/Scenes/Artwork/Components/CommercialButtons/MakeOfferButton.tsx [hunk 2]
```diff
@@ -48,6 +49,7 @@ export class MakeOfferButton extends React.Component<MakeOfferButtonProps, State
     context_module: Schema.ContextModules.CommercialButtons,
   })
   handleCreateOfferOrder() {
+    console.log("handling make offer")
```
Comment (sweir27) [inline src/lib/Scenes/Artwork/Components/CommercialButtons/MakeOfferButton.tsx]: 👋

File: src/lib/AppRegistry.tsx (hunk 1 above)
Comment (kierangillen) [reply src/lib/AppRegistry.tsx]: It's a Prop that's being passed to the Artwork view: https://github.com/artsy/emission/blob/master/src/lib/Scenes/Artwork/Artwork.tsx#L289

File: src/lib/Scenes/Artwork/Components/CommercialButtons/MakeOfferButton.tsx (hunk 2 above)
Comment (kierangillen) [reply src/lib/Scenes/Artwork/Components/CommercialButtons/MakeOfferButton.tsx]: Refresh :)

**Pull request merged**